src/Native/Unix/System.Security.Cryptography.Native.Apple/pal_ssl.cpp

```diff
@@ -363,7 +363,7 @@ extern "C" int32_t AppleCryptoNative_SslGetProtocolVersion(SSLContextRef sslCont
         return errSecParam;
 
     SSLProtocol protocol = kSSLProtocolUnknown;
-    int osStatus = SSLGetNegotiatedProtocolVersion(sslContext, &protocol);
+    OSStatus osStatus = SSLGetNegotiatedProtocolVersion(sslContext, &protocol);
```

Comment:
Yeah.  If it ever changes we'll get compiler type-safety errors; but it's still good to be honest about what's what.